B827EBFFFE540956.json

```diff
@@ -0,0 +1,18 @@
+{
+  "gateway_conf": {
+    "gateway_ID": "b827ebfffe540956",
```

Comment:
This ID should be uppercase, otherwise it won't work. Please update and then we'll merge.